sdk/core/perf/src/base_test.cpp

```diff
@@ -151,7 +150,10 @@ namespace Azure { namespace Perf {
 
       //  Make one call to Run() before starting recording, to avoid capturing one-time setup
       //  like authorization requests.
-      this->Run(ctx);
+      if (!m_options.ShouldSkipInitial())
+      {
+        this->Run(ctx);
+      }
```

Comment:
Again, why ... not obvious.
You see how it works with the comment above, now it is more confusing.
```
//  Make one call to Run() before starting recording, to avoid capturing one-time setup
//  like authorization requests.
```
And when the nest line was just `this->Run(ctx);`, it was not questionable.

But now, there is an "if", and I kind of scratch my head.